heron/api/src/java/com/twitter/heron/streamlet/Config.java

```diff
@@ -65,8 +65,10 @@ public static Config defaultConfig() {
 
   public static class Builder {
     private com.twitter.heron.api.Config config;
+    private boolean useKryo;
```

Comment:
Hmm. Do we really want to make Kryo as default?

Also I don't like this new boolean. If in future we need/have a new serializer, this code could become messy. In fact it already has issues If user calls: builder. useKryoSerializer(false). useKryoSerializer().

I am wondering if an enum could be cleaner here.

Reply:
I conferred with @srkukarni on this and he agreed that the performance improvements associated with using Kryo are significant enough (with no apparent drawbacks) that Kryo should be the default. As for an enum, that could make things trickier in the future when enabling people to use any serializer they'd like.